Improve question wording (#152)
common-workflow-language/user_guide: _episodes/02-1st-example.md

teaching: 5
exercises: 0
questions:

> - "How do I write a CWL wrapper for a simple command line tool?"

Reviewer: ```suggestion
- "How do I write a CWL description of a simple command line tool?"
```
